aframe-audio-components/join-room.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Join Room - Virtual Electronic Music</title>
+    <meta name="description" content="Join Room - Virtual Electronic Music">
+    <!-- Bootstrap CDN links -->
+     <!-- Latest compiled and minified CSS -->
+     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+
+     <!-- jQuery library -->
+     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+ 
+     <!-- Latest compiled JavaScript -->
+     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+
+     <!-- CSS for this page -->
+    <link rel="stylesheet" type="text/css" href="content/styles/join-room-style.css" >
+    
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
```

Comment:
I don't think any of the osc or aframe scripts are needed in this html and will be outdated soon anyways with the osc-rework branch

Reply:
got it, just removed!